docs/user_guide.md

```diff
@@ -153,6 +153,105 @@ Setting up tradeable, collateralized markets
     - **Price** : `500`
     - **Quantity** : `1.0`
 
+## Setup Clearinghouse
+52. Login as `Ccp`
+53. `Ccp`: Go to Wallet, create a regular account
+    - **Provider**: `Bank`
+    - **Account Name** : `Clearing-Bank`
+    - **Account Type** : `Regular`
+54. `Ccp`: Go to Manage/Clearing and "Accept" Clearing Role
+    - **Clearing Account**: `Clearing-Bank`
+55. Login as `Alice`
+56. `Alice`: Go to Wallet, create a regular account
+    - **Provider**: `Bank`
+    - **Account Name** : `ClearingAlice-Bank`
+    - **Account Type** : `Regular`
+57. `Alice`: Go to Wallet, create an allocation account
+    - **Provider**: `Bank`
+    - **Account Name** : `MarginAlice-Bank`
+    - **Account Type** : `Allocation`
+    - **Nominee** : `Ccp`
+58. `Alice`: On Landing, click "Request Clearing Service"
+    - **Provider**: `CCP`
+    - **Clearing Account**: `ClearingAlice-Bank`
+    - **Margin Account**: `MarginAlice-Bank`
+59. `Alice`: Go to Wallet, click on row for `ClearingBob-Bank`, Deposit 10,000 USD
+60. Login as `Bob`
+61. `Bob`: Go to Wallet, create a regular account
+    - **Provider**: `Bank`
+    - **Account Name** : `ClearingBob-Bank`
+    - **Account Type** : `Regular`
+62. `Bob`: Go to Wallet, create an allocation account
+    - **Provider**: `Bank`
+    - **Account Name** : `MarginBob-Bank`
+    - **Account Type** : `Allocation`
+    - **Nominee** : `Ccp`
+63. `Bob`: On Landing, click "Request Clearing Service"
+    - **Provider**: `CCP`
+    - **Clearing Account**: `ClearingBob-Bank`
+    - **Margin Account**: `MarginBob-Bank`
+64. `Bob`: Go to Wallet, click on row for `ClearingBob-Bank`, Deposit 10,000 USD
+
+### Test Margin Calls
+Perform successful margin call for Alice
+
+65. Login as `Ccp`
+66. `Ccp`: on `Members` page, click "Perform Margin Call":
+    - **Customer**: `Alice`
+    - **Amount**: 5000
+
+Fail and retry a margin calculation for `Bob`
+67. `Ccp`: on `Members` page, click "Perform Margin Call":
```

Comment:
Should have a new line before `67.`, otherwise markdown renders this all on one paragraph